src/Rules/Registry.php

```diff
@@ -6,11 +6,21 @@ class Registry
 {
 
 	/**
-	 * @var \PHPStan\Rules\Rule[]
+	 * @var \PHPStan\Rules\Rule[][]
 	 */
 	private $rules;
 
-	public function register(Rule $rule)
+	/**
+	 * @param \PHPStan\Rules\Rule[] $rules
+	 */
+	public function __construct(array $rules)
+	{
+		foreach ($rules as $rule) {
+			$this->register($rule);
+		}
+	}
+
+	private function register(Rule $rule)
```

Comment:
More ways to adjust class behavior usually leads to more bugs.
I'd recommend using one and adding another WHEN really needed.